### DISCUSSION TITLE
Add ENV from docker-compose in to openid section in recipe

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#13705] [Q&A] [answered]
I would like to add Add ENV from docker-compose in to openid section in recipe.
Any idea how to do it in the fast and easy way?

**Comment by ns8482e** [comment, marked as answer]
Nice question! Orchard already supports reading configuration items from recipe.

For example you can call `[js: configuration('MyConfigSec:MyConfigKey', '')]` to read value from config for current tenant. Documented here https://docs.orchardcore.net/en/main/docs/reference/modules/Setup/#recipe-configuration-keys

You can also provide any config value from env variables in asp.net out of the box.

So here you can configure docker-compose env key as `OrchardCore__MyConfigSec__MyConfigKey` (for all tenants) OR `OrchardCore__Default__MyConfigSec__MyConfigKey` (for specific tenant)
